src/Search_Replace_Command.php

```diff
@@ -108,7 +116,50 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * : Produce report. Defaults to true.
 	 *
 	 * [--report-changed-only]
-	 * : Report changed fields only. Defaults to false.
+	 * : Report changed fields only. Defaults to false, unless logging, when it defaults to true.
+	 *
+	 * [--log[=<file>]]
+	 * : Log the items changed. If <file> is not supplied or is "-", will output to STDOUT.
+	 * Warning: causes a significant slow down, similar or worse to enabling --precise or --regex.
+	 *
+	 * [--log_context=<before_num,after_num>]
+	 * : Number of characters to display before and after the match. One number sets both before and after. Two comma-separated numbers set before and after respectively. Defaults to 40. Ignored if not logging.
+	 *
+	 * [--log_prefixes=<old_prefix,new_prefix>]
+	 * : The old and new prefixes to prepend to the log lines for old match and new replacement, comma-separated. Defaults to '< ,> '. Use ',' for no prefixes. Ignored if not logging.
+	 *
+	 * [--log_colors=<table_column_id_color,old_color,new_color>]
```

Comment:
Same comment as above: does this need to be a command argument?

I think my preference would be to remove this as an option entirely. We can bring it up again in the future if anyone brings it up.

Reply:
Okay, will do.